server/.gitignore

```diff
@@ -1,2 +1,4 @@
 config.json
 *.pyc
+settings.json
+.gitignore
```

Comment:
```suggestion

```
Don't gitignore `.gitignore`, or it won't be included! Also, you're missing a trailing newline.

Reply:
But I thought I needed to ignore `.gitignore`, otherwise I'm gonna have things like `settings.json` in there which you don't need